torch/quantization/fake_quantize.py

```diff
@@ -154,18 +155,30 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                                                       dtype=torch.quint8,
                                                       qscheme=torch.per_tensor_affine,
                                                       reduce_range=True)
+
+def _is_fake_quant_script_module(mod):
+    ''' Returns true if given mod is an instance of FakeQuantize script module.
+    '''
+    if isinstance(mod, torch.jit.RecursiveScriptModule):
+        # qualified name looks like '__torch__.torch.quantization.fake_quantize.___torch_mangle_2.FakeQuantize'
+        suffix = mod._c.qualified_name.split('.', 1)[1]
+        name = re.sub(r'\.___torch_mangle_\d+', '', suffix)
+        return name == 'torch.quantization.fake_quantize.FakeQuantize'
+    else:
+        return false
```

Comment:
nit: this can be `return False`(without else), also it's `False` instead 'false` I think.